ktor-core/src/org/jetbrains/ktor/application/ApplicationResponse.kt

```diff
@@ -12,6 +13,20 @@ public interface ApplicationResponse {
     public fun status(value: HttpStatusCode)
     public fun interceptStatus(handler: (value: HttpStatusCode, next: (value: HttpStatusCode) -> Unit) -> Unit)
 
+    public fun cookie(name: String): Cookie? = headers.values("Set-Cookie").map { parseServerSetCookieHeader(it) }.firstOrNull { it.name == name }
+    public fun cookie(item: Cookie): Unit = headers.append("Set-Cookie", renderSetCookieHeader(item))
```

Comment:
what if we implement it like we did for headers? (so it will look like `response.cookies.append`)